server/tests/2-recipes.js

```diff
@@ -28,24 +28,24 @@ describe('/POST Create User', () => {
 });
 
 describe('/GET all Recipes Test', () => {
-  it('should return an empty array of Recipes', (done) => {
+  it('should return a \'not found\'', (done) => {
     chai.request(server)
```

Comment:
Use more description it blocks on all test spec

Reply:
Okay